src/crypto/bcrypt/base64.cr

```diff
@@ -20,7 +20,7 @@ module Crypto::Bcrypt::Base64
     51, 52, 53, -1, -1, -1, -1, -1,
   ]
 
-  def self.encode(d, len) : String
+  def self.encode(d : Slice(UInt8) | Array(UInt8), len : Int32) : String
```

Comment:
The type of `d` looks odd. Perhaps it should just be `Indexable(UInt8)` ? Or does it accept strings as well?

Reply:
I believe this parameter should in fact only accept `Slice(UInt8)`.
`Array(UInt8)` comes from a unit test, which should use a slice instead.

It's also an internal helper method, so type restrictions might are not really necessary anyway. Ditto below.

Reply:
Cool, I'll remove the restrictions then.